api/formatter.py

```diff
@@ -0,0 +1,7 @@
+def formatter(game):
+    
+    game = game.replace("'","").replace("-"," ").replace(",","").replace(".","").lower()
```

Comment:
The usual way for two people to work together in the same branch would be via PRs. For example, @JPBP22 started a data prep process, now instead of @HenningGC committing to the same branch, he would do a PR to Jose's branch. If Jose is ok with the code, he would merge the code into his branch, and will ask the rest of the team to review before emerging to main.

Makes sense?